File: javaScript/fizzBuzz.js
out = ""

for (i = 1; i <= 100; i++) {
    if (i % 3 == 0 && i % 5 == 0) {
        out += "FizzBuzz "
    }
    else if(i % 3 == 0) {
        out += "Fizz "
    }
    else if (i % 5 == 0) {
        out += "Buzz "
    }
    else {
        out += i + " "
    }
}

console.log(out)

File: Python/fizzBuzz.py
out = ""
for i in range(1, 101):
    if i % 15 == 0:
        out += "FizzBuzz "
    elif i % 3 == 0:
        out += "Fizz "
    elif i % 5 == 0:
        out += "Buzz "
    else:
        out += str(i) + " "
print(out)